app/views/content/subjects/physics/_article.html.erb

```diff
@@ -134,7 +134,7 @@
   <section class="col col-space-m col-720">
     <h3>Fund your training to teach physics</h3>
     <p>
-      Scholarships of £29,000 are available from the Institute of Physics. Alternatively, bursaries of £27,000 are available for trainee physics teachers starting their teacher training course from September 2023.
+      You could receive a scholarship of £29,000 from the Institute of Physics or a bursary of £27,000, if you start your teacher training course between September 2023 and July 2024.
```

Comment:
I think this sounds a bit like if you start it in Sept 2024, you definitely won't get anything?

Last year we also added this - do we need something similar?
Financial support for courses starting in 2023 will be announced in autumn 2022.

(see https://github.com/DFE-Digital/get-into-teaching-app/pull/2837)